src/View/Definition.elm

```diff
@@ -165,105 +164,17 @@ splitOnEndKeyword partial =
 categoryListStyle : Style
 categoryListStyle =
     batch
-        [ margin2 (rem 2) zero ]
+        [ margin2 (rem 2) (rem 1) ]
 
 
 introStyle : Style
 introStyle =
     batch
-        [ color purple
-        , fontSize (rem 1)
-        , margin2 (rem 2) zero
-        ]
+        [ margin2 (rem 2) (rem 1) ]
 
 
 keywordStyle : Style
 keywordStyle =
     batch
         [ backgroundColor lightGreen
         ]
-
-
-expanderButtonStyle : Style
```

Comment:
I guess you've decided these will remain identical? I had small hopes those 2 pages would get different personalities.

Reply:
They are the same component, atm so I think easier to look after in one place. It's easy enough to override styles with a more specific one in a page, if we have a need.